Documentation/MRTK_PackageContents.md

```diff
@@ -13,20 +13,20 @@ The Microsoft.MixedRealityToolkit.Unity.Foundation package includes the core com
 
 | Folder | Component | Description |
 | --- | --- | --- |
-| MixedRealityToolkit | | Interface and type definitions, base classes, standard shader. |
-| MixedRealityToolkit.Providers | | |
+| MRTK\Core | | Interface and type definitions, base classes, standard shader. |
+| MRTK\Providers | | |
```

Comment:
Love that you were using the forward slash for paths everywhere to be consistent and "cross platform". Does the forward slash not work in markdown tables?

Reply:
it should. i'll flip them